operations/README.md

```diff
@@ -4,42 +4,43 @@ The Cray System Management (CSM) operational activities are administrative proce
-- [HPE Power Distribution Unit (PDU)](#hpe-power-distribution-unit-pdu)
-- [Node management](#node-management)
-- [Network](#network)
-      - [Management network](#management-network)
-      - [Customer accessible networks (CMN/CAN/CHN)](#customer-accessible-networks-cmncanchn)
-      - [Dynamic Host Configuration Protocol (DHCP)](#dynamic-host-configuration-protocol-dhcp)
-      - [Domain Name Service (DNS)](#domain-name-service-dns)
-      - [External DNS](#external-dns)
-      - [MetalLB in BGP-mode](#metallb-in-bgp-mode)
-- [Spire](#spire)
-- [Update firmware with FAS](#update-firmware-with-fas)
-- [System Admin Toolkit (SAT)](#system-admin-toolkit-sat)
-- [Install and Upgrade Framework (IUF)](#install-and-upgrade-framework-iuf)
-- [Backup and recovery](#backup-and-recovery)
-- [Multi-tenancy](#multi-tenancy)
+- [Cray System Management (CSM) Administration Guide](#cray-system-management-csm-administration-guide)
+    - [CSM product management](#csm-product-management)
+    - [Bare-metal](#bare-metal)
+    - [Image management](#image-management)
+    - [Boot orchestration](#boot-orchestration)
+    - [System power off procedures](#system-power-off-procedures)
+    - [System power on procedures](#system-power-on-procedures)
+    - [Power management](#power-management)
+    - [Artifact management](#artifact-management)
+    - [Compute rolling upgrades](#compute-rolling-upgrades)
+    - [Configuration management](#configuration-management)
+    - [Kubernetes](#kubernetes)
+    - [Package repository management](#package-repository-management)
+    - [Security and authentication](#security-and-authentication)
+    - [Resiliency](#resiliency)
+    - [ConMan](#conman)
+    - [Utility storage](#utility-storage)
+    - [System management health](#system-management-health)
+    - [System Layout Service (SLS)](#system-layout-service-sls)
+    - [System configuration service](#system-configuration-service)
+    - [Hardware State Manager (HSM)](#hardware-state-manager-hsm)
+    - [Hardware Management (HM) collector](#hardware-management-hm-collector)
+    - [HPE Power Distribution Unit (PDU)](#hpe-power-distribution-unit-pdu)
+    - [Node management](#node-management)
+    - [Network](#network)
+        - [Management network](#management-network)
+        - [Customer accessible networks (CMN/CAN/CHN)](#customer-accessible-networks-cmncanchn)
+        - [Dynamic Host Configuration Protocol (DHCP)](#dynamic-host-configuration-protocol-dhcp)
+        - [Domain Name Service (DNS)](#domain-name-service-dns)
+        - [External DNS](#external-dns)
+        - [MetalLB in BGP-mode](#metallb-in-bgp-mode)
+    - [Spire](#spire)
+    - [Update firmware with FAS](#update-firmware-with-fas)
+    - [System Admin Toolkit (SAT)](#system-admin-toolkit-sat)
+    - [Install and Upgrade Framework (IUF)](#install-and-upgrade-framework-iuf)
+    - [Backup and recovery](#backup-and-recovery)
+    - [Multi-tenancy](#multi-tenancy)
```

Comment:
```suggestion
- [CSM product management](#csm-product-management)
- [Bare-metal](#bare-metal)
- [Image management](#image-management)
- [Boot orchestration](#boot-orchestration)
- [System power off procedures](#system-power-off-procedures)
- [System power on procedures](#system-power-on-procedures)
- [Power management](#power-management)
- [Artifact management](#artifact-management)
- [Compute rolling upgrades](#compute-rolling-upgrades)
- [Configuration management](#configuration-management)
- [Kubernetes](#kubernetes)
- [Package repository management](#package-repository-management)
- [Security and authentication](#security-and-authentication)
- [Resiliency](#resiliency)
- [ConMan](#conman)
- [Utility storage](#utility-storage)
- [System management health](#system-management-health)
- [System Layout Service (SLS)](#system-layout-service-sls)
- [System configuration service](#system-configuration-service)
- [Hardware State Manager (HSM)](#hardware-state-manager-hsm)
- [Hardware Management (HM) collector](#hardware-management-hm-collector)
- [HPE Power Distribution Unit (PDU)](#hpe-power-distribution-unit-pdu)
- [Node management](#node-management)
- [Network](#network)
    - [Management network](#management-network)
    - [Customer accessible networks (CMN/CAN/CHN)](#customer-accessible-networks-cmncanchn)
    - [Dynamic Host Configuration Protocol (DHCP)](#dynamic-host-configuration-protocol-dhcp)
    - [Domain Name Service (DNS)](#domain-name-service-dns)
    - [External DNS](#external-dns)
    - [MetalLB in BGP-mode](#metallb-in-bgp-mode)
- [Spire](#spire)
- [Update firmware with FAS](#update-firmware-with-fas)
- [System Admin Toolkit (SAT)](#system-admin-toolkit-sat)
- [Install and Upgrade Framework (IUF)](#install-and-upgrade-framework-iuf)
- [Backup and recovery](#backup-and-recovery)
- [Multi-tenancy](#multi-tenancy)
```

Reply:
This can be handled with a different ticket.

Reply:
Fixed.